ext/native.go

```diff
@@ -155,6 +156,14 @@ func (tp *nativeTypeProvider) FindStructType(typeName string) (*types.Type, bool
 	return tp.baseProvider.FindStructType(typeName)
 }
 
+func toFieldName(f reflect.StructField) string {
+	if name, found := f.Tag.Lookup("cel"); found {
```

Comment:
Please document the tag being searched with an example in the top-level comments. Also make sure these comments are in sync with the README.md

Reply:
Do you have a specific place in mind in the README? It doesn't appear as if native is mentioned at all 😅

Reply:
Ah, then ignore the README.md comment. In the local file is fine. Thanks!